src/uct/ib/base/ib_iface.c

```diff
@@ -190,19 +190,30 @@ int uct_ib_iface_is_reachable(const uct_iface_h tl_iface, const uct_device_addr_
     uct_ib_iface_t *iface = ucs_derived_of(tl_iface, uct_ib_iface_t);
     const uct_ib_address_t *ib_addr = (const void*)dev_addr;
     union ibv_gid gid;
-    uint8_t is_global, is_local_ib = 1;
+    uint8_t is_global;
     uint16_t lid;
+    int is_local_ib = 0;
```

Comment:
the default case should exit with fatal error so this var would not be unused.
as general rule it's better to avoid initialization of vars to let compiler/coverity/valgrind to discover potential errors